docs/trino-connector/supported-catalog.md

```diff
@@ -15,7 +15,43 @@ The catalogs currently supported by the Gravitino connector are as follows:
 
 ## Create catalog
 
-Trino currently does not support creating Gravitino managed catalogs, if you need to create a catalog, please refer to: [Create a Catalog](../manage-metadata-using-gravitino.md#create-a-catalog).
+Trino itself does not support creating catalogs. 
+Users can create catalogs through the Gravitino connector and then load them into Trino. 
+The Gravitino connector provides the following stored procedures to create and delete catalogs.
+
+Create catalog:
+
+```text
+createCatalog(CATALOG varchar, PROVIDER varchar, PROPERTIES varchar, IGNORE EXIST boolean);
+```
+
+- CATALOG: The catalog name to be created.
+- PROVIDER: The catalog provider, currently only supports `hive`, `lakehouse-iceberg`, `jdbc-mysql`, `jdbc-postgresql`.
+- PROPERTIES: The properties of the catalog, the format is a json string. like `{"key1":"value1","key2":"value2"}`.
```

Comment:
Should you also specify which catalog properties are supported or provided by referring to the specific docs?

Reply:
Also can you follow this Iceberg doc to make it as a table for properties. https://iceberg.apache.org/docs/latest/spark-procedures/